src/locales/en.js

```diff
@@ -0,0 +1,32 @@
+      "COVID-19 is the new respiratory disease spreading around the world and it is caused by a coronavirus.",
+    protection: "How to protect yourself",
+    protection_detail:
+      "There is currently no vaccine to prevent coronavirus disease 2009 (COVID-19).",
+    sick: "What to do if you are sick",
+    sick_detail:
+      "A guide to help you make decisions and seek appropriate medical care.",
+    care: "How to care for someone",
+    care_detail:
+      "Monitor the person for worsening symptoms, know the emergency warning signs.",
+    community_form: "COMMUNITY FORM",
+    passenger_form: "PASSENGER FORM",
+    public_report: "PUBLIC REPORT",
+    volunteer_form: "VOLUNTEER FORM"
+  },
+  care:
+    "Monitor the person for worsening symptoms, know the emergency warning signs. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+  protection:
+    "There is currently no vaccine to prevent coronavirus disease 2009 (COVID-19). Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+  sick:
+    "A guide to help you make decisions and seek appropriate medical care. COVID-19 is the new respiratory disease spreading around the world and it is caused by a coronavirus. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+  what:
+    "COVID-19 is the new respiratory disease spreading around the world and it is caused by a coronavirus. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```

Comment:
I decided to convert the trans files to .js so that we can import the lengthy detail contents from another file later.